tasks/create-pyxis-image/tests/mocks.sh

```diff
@@ -61,6 +61,17 @@ function oras() {
   if [[ "$*" == "manifest fetch --registry-config"* ]]
   then
     echo '{"mediaType": "my_media_type"}'
+  elif [[ "$*" == "pull --registry-config"*dockerfile-not-found* ]]
```

Comment:
Shouldn't these `dockerfile-not-found` strings be in the tests somewhere?

Reply:
Good catch! I wrote two new tests, but forgot to `git add` them. Fixed.